src/servers/authorization/email.ts

```diff
@@ -0,0 +1,135 @@
+import express from "express"
+
+import { isProd } from "@config"
+
+import { recordExceptionInCurrentSpan, wrapAsyncToRunInSpan } from "@services/tracing"
+
+import { parseIps } from "@domain/accounts-ips"
+
+import { checkedToEmailAddress } from "@domain/users"
+import { loginWithEmail, requestEmailCode } from "@app/authentication"
+
+import { parseErrorMessageFromUnknown } from "@domain/shared"
+
+import { checkedToEmailCode } from "@domain/authentication"
+import { checkedToFlowId } from "@services/kratos"
+
+import { baseLogger } from "@services/logger"
+
+import {
+  EmailCodeInvalidError,
+  EmailValidationSubmittedTooOftenError,
+} from "@domain/authentication/errors"
+
+import { UserLoginIpRateLimiterExceededError } from "@domain/rate-limit/errors"
+
+import { authRouter } from "./router"
+
+// TODO: should code request behind captcha or device token?
+authRouter.post(
+  "/email/code-request",
```

Comment:
In http the verb maps to  the http operation and the path maps to the resource. perhaps use `/email/auth-code`?